docs/LTS.md

```diff
@@ -13,7 +13,15 @@ out in this document:
    [https://github.com/fastify/fastify/releases](https://github.com/fastify/fastify/releases).
 
 1. Major releases will receive security updates for an additional six months
-   from the release of the next major release.
+   from the release of the next major release. After this period is expired,
+   we will still review and release security fixes as long as they are
+   provided by the community and they do not violate other constrains,
```

Comment:
"contrains" => "contraints"

Reply:
Whoops, looks like you also have a typo @jsumners
"contraints" => "constraints"

Reply:
🤣